flowapi/flowapi/qa_endpoints.py

```diff
@@ -0,0 +1,205 @@
+from quart_jwt_extended import jwt_required, current_user
+from quart import Blueprint, current_app, request
+import datetime as dt
+
+blueprint = Blueprint("qa", __name__)
+
+
+@blueprint.route("/<cdr_type>/<check_id>", methods=["GET"])
+@jwt_required
+async def get_qa_date_range(cdr_type, check_id):
+    """
+    Returns QA values for a given check type and call id between two dates
+    ---
+    get:
+      parameters:
+      - name: cdr_type
+        in: path
+        required: true
+        schema:
+          type: string
+      - name: check_id
+        in: path
+        required: true
+        schema:
+          type: string
+      - name: start_date
+        required: true
+        schema:
+          type: string
+          format: date
+        in: query
+      - name: end_date
+        required: true
+        schema:
+          type: string
+          format: date
+        in: query
+      responses:
+        '200':
+          description: Dates available for each event type.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  qa_checks:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        outcome:
+                          type: string
+                        type_of_query_or_check:
+                          type: string
+                        cdr_date:
+                          type: string
+                          format: date
+        '400':
+          description: Bad request
+        '401':
+          description: Unauthorized.
+        '404':
+          description: No QA checks of type specified found on date
+        '500':
+          description: Server error.
+      summary: Returns QA values for a given check type and call id between two dates
+    """
+    current_user.can_get_qa()
+    return await get_qa_checks(
+        cdr_type, check_id, request.args.get("start_date"), request.args.get("end_date")
```

Comment:
I don't think its a huge issue really, but it struck me while thinking that one could respond to a hit on this with no params by checking first/last in available dates that these endpoints implicitly reveal available dates as well, so perhaps one wants to mandate that scope is provided as well? (I'd say maybe not, except there's not really a mechanism for us to make clear that you in effect have access to the same information)